### PR TITLE
fix: font style fixed

### PR DESCRIPTION
**Pull request by SourabhKanade**
Now font looks good.

image

### CODE REVIEW

File: components/homepage/Footer/Footer.jsx [hunk 1]
```diff
@@ -30,7 +30,10 @@ const Footer = () => {
                   />
                 </div>
                 <div className={style.message}>
-                  <textarea
+                  <textarea style={{
+                    fontSize: "15px",
```
Comment (kaustubhai) [inline components/homepage/Footer/Footer.jsx]: Why have you choose inline styling here when you have done similar thing in the CSS as well?
Also, please go through the CONTRIBUTING.md once before committing changes again

**Review by kaustubhai — CHANGES_REQUESTED** [review]
Also please remove `package.json` and `package-lock.json` in your commits

File: components/homepage/Footer/Footer.jsx (hunk 1 above)
Comment (SourabhKanade) [reply components/homepage/Footer/Footer.jsx]: Okay

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+/package-lock.json
+/package.json
```
Comment (kaustubhai) [inline .gitignore]: what if someone actually uses a new package in the code, this can't be here

**Review by kaustubhai — CHANGES_REQUESTED** [review]
You deleted `package.json` and `package-lock.json`, if I'l merge this as it is, I'll lose these files. Please instead of deleting, restore to them as in the last merged commit

**Comment by SourabhKanade** [comment]
@kaustubhai I hope now everything is okay?

**Comment by kaustubhai** [comment]
>  Please instead of deleting, restore to them as in the last merged commit
I will still lose my `package.json` and `package-lock.json`

You should restore them to thier last correct version

**Comment by kaustubhai** [comment]
@SourabhKanade can you please share Issue number here, thanks.

**Comment by SourabhKanade** [comment]
> @SourabhKanade can you please share Issue number here, thanks.

Hello @kaustubhai this was issue number #63 
THANKS 👍

**Pull request merged**